ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPage.java

```diff
@@ -20,7 +24,11 @@
 	SELECT_START_CONNECTION(new IntegrationConnectionSelectComponentStart()),
 	SELECT_FINISH_CONNECTION(new IntegrationConnectionSelectComponentFinish()),
 	CHOOSE_A_FINISH_CONNECTION(new IntegrationConnectionSelectComponentFinish()),
-	ADD_TO_INTEGRATION(new IntegrationSaveOrAddStepComponent());
+	ADD_TO_INTEGRATION(new IntegrationSaveOrAddStepComponent()),
+	CUSTOMIZATIONS(new CustomizationsPage()),
+	TECHNICAL_EXTENSIONS(new TechExtensionsListComponent()),
+	IMPORT_TECHNICAL_EXTENSIONS(new TechExtensionsImportPage()),
```

Comment:
Maybe rename `IMPORT_TECHNICAL_EXTENSIONS` to `TECHNICAL_EXTENSIONS_IMPORT` to keep the pattern.